src/chromo/decay_afterburner.py

```diff
@@ -0,0 +1,96 @@
+import chromo
+import numpy as np
+
+
+class DecayAfterburner:
```

Comment:
What happens if this is imported on Windows where Pythia8 is not built?

Reply:
It will raise an exception that the library is now found. I haven't checked, but I guess so based on errors when Pythia8 tests are not skipped on CI.

Reply:
It should raise "ModuleNotFoundError".